aom/src/main/java/com/nedap/archie/adl14/ADL2ConversionResult.java

```diff
@@ -18,24 +18,13 @@ public class ADL2ConversionResult {
     private MessageLogger log;
     private Exception exception;
 
-    /**
-     * empty constructor for JSON parsing. Do not use
-     */
     public ADL2ConversionResult() {
-
+        /* Empty construction for Jackson parsing */
```

Comment:
You have moved javadoc comments, which are actually converted to documentation on the build, to an internal comment. That does not seem like a good idea, as the javadoc will no longer list that this constructor should not be used.